peppy/sample.py

```diff
@@ -15,19 +14,23 @@
 from pandas import isnull, Series
 import yaml
 
-from . import ASSAY_KEY, SAMPLE_NAME_COLNAME
 from attmap import AttMap, PathExAttMap
-from .const import \
-    ALL_INPUTS_ATTR_NAME, DATA_SOURCE_COLNAME, DATA_SOURCES_SECTION, \
-    NAME_TABLE_ATTR, REQUIRED_INPUTS_ATTR_NAME, SAMPLE_EXECUTION_TOGGLE, \
-    SAMPLE_SUBANNOTATIONS_KEY, VALID_READ_TYPES
-from .utils import check_bam, check_fastq, copy, get_file_size, \
+from .const import *
+from .const import SNAKEMAKE_SAMPLE_COL
+from .utils import copy, get_file_size, get_logger, get_name_depr_msg, \
     grab_project_data, parse_ftype, sample_folder
+from ngstk import peek_read_lengths_and_paired_counts_from_bam
```

Comment:
the new ngstk is showing up here already!

 that is quite a function name!

Reply:
wow you've already got it on pypi. cool.

Reply:
I couldn't come up with anything both meaningful and short...`check_bam` made a lot of sense when used more locally, but I wanted a more descriptive name for export from the seq. utils package

Reply:
OK https://github.com/pepkit/peppy/pull/301/commits/1ee00259c94315f422da1ad1c5efe8e99a74d1c0 notes some improvements and changes